arm/Microsoft.DataFactory/factories/readme.md

```diff
@@ -35,6 +37,21 @@
 | `tags` | object | `{object}` |  | Optional. Tags of the resource. |
 | `workspaceId` | string |  |  | Optional. Resource identifier of Log Analytics. |
 
+
+### Parameter Usage: [`integrationRuntime`](https://docs.microsoft.com/en-us/azure/templates/microsoft.datafactory/factories/integrationruntimes?tabs=bicep)
```

Comment:
Can be removed. If you generate the ReadMe with the function, the parameter should get a link in the above parameters table type column. Please doublecheck

Reply:
Unfortunately, for some reason the ReadMe script does now work for me. But that's a conversation for another thread :)
![image](https://user-images.githubusercontent.com/47188710/142948690-5cb02498-5b49-404f-8ebc-902d748f6c09.png)

I have gone and updated manually based on what's in other modules

Reply:
Added a minor update in PR : https://github.com/Azure/ResourceModules/pull/529